styx-scheduler-service/src/test/java/com/spotify/styx/docker/LabelValueTest.java

```diff
@@ -53,6 +53,14 @@ public void shouldNotModifySimple63ByteValue() {
     assertThat(output, is(input));
   }
 
+  @Test
+  public void shouldPadInvalidFirstChar() {
+    var input = "-spotify#com";
```

Comment:
Please remove `#` to make this a test for a single concern
```suggestion
    var input = "-spotify";
```